src/app/datapoints-graph/datapoints-graph-config/datapoints-graph-widget-config.component.html

```diff
@@ -47,6 +41,31 @@
         <label>{{ 'Display options' | translate }}</label>
         <fieldset class="c8y-fieldset m-b-24 m-t-0">
           <c8y-form-group class="p-b-16 m-b-0 p-t-8">
+            <label
+              [title]="'Enable date selection in the widget view.' | translate"
+              class="c8y-checkbox"
+            >
+              <input
+                type="checkbox"
+                formControlName="displayDateSelection"
+                name="displayDateSelection"
+              />
+              <span></span>
+              <span translate>Date selection in the widget view.</span>
+              <button
```

Comment:
The padding of this element is somehow off. Can we make it more dense?
![image](https://github.com/user-attachments/assets/3ced0189-41c3-446f-9168-ba20c79e8e84)